### ISSUE TITLE
Some MDN pages are overflowing the viewport width

### ISSUE THREAD

**Issue by vkWeb** [mdn/yari#2141]
I observed this while visiting some random pages to see how our new update looks: https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/HTML_text_fundamentals

Screenshot:
![image](https://user-images.githubusercontent.com/26724128/102144742-067b0080-3e8c-11eb-8c38-ab30552eab1e.png)

**Comment by tannerdolby** [comment]
Yes, I can see this overflow issue as well. It looks to be happening for viewports greater than 1200px. Everything under 1200px within the main grid looks good.

To address the page overflow problem, it seems that the media query for the main grid container `<div class="page-wrapper document-page">` is having its `grid-template-columns` property overridden (as it should) when the viewport is greater than `74.9375em` to be `grid-template-columns: 100%`. 

Screenshot:
<img width="504" alt="Screen Shot 2020-12-15 at 12 45 55 AM" src="https://user-images.githubusercontent.com/48612525/102191963-f7d22080-3e6e-11eb-9e48-b21a688c3ddd.png">


When I toggle off the `grid-template-columns: 100%` styling for viewports greater than 1200px, the overflow issue goes away and the page no longer has any scroll mechanism as there isn't any content overflowing.

**Comment by zcdunn** [comment]
Clicking the _Feedback_ menu on a viewport of 1090px causes the menu to overflow.

**Comment by mattiapontonio** [comment]
https://github.com/mdn/sprints/issues/3969

**Comment by mattiapontonio** [comment]
@zcdunn can You screenshot?

**Comment by zcdunn** [comment]
Here is the open _Feedback_ menu overflowing
![image](https://user-images.githubusercontent.com/7121969/102903232-08216700-443e-11eb-813c-76384ce2ec41.png)

**Comment by mattiapontonio** [comment]
For the _dropdown_ I recently discovered
https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API

I suppose that You can check if overflow and then set `left` or `right` values.

Otherwise in this case the _dropdown_ should be always aligned to the `right` of the parent in a `relative` `position`.

**Comment by mattiapontonio** [comment]
> Here is the open _Feedback_ menu overflowing
> ![image](https://user-images.githubusercontent.com/7121969/102903232-08216700-443e-11eb-813c-76384ce2ec41.png)
> 

For the _dropdown_ I recently discovered
https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API

I suppose that You can check if overflow and then set `left` or `right` values.

Otherwise in this case the _dropdown_ should be always aligned to the `right` of the parent in a `relative` `position`.

**Comment by mattiapontonio** [comment]
https://github.com/mdn/yari/issues/2233

**Comment by schalkneethling** [comment]
@vkWeb Can you confirm that this is now fixed? Thank you.

**Comment by zcdunn** [comment]
I can confirm that the menu no longer overflows, but some content still does. The textarea at https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/HTML_text_fundamentals#active_learning_marking_up_our_recipe_page is one example

**Comment by vkWeb** [comment]
@schalkneethling I can confirm that this issue has been resolved. There is no content / page I found to be overflowing. :tada: 

@zcdunn umm, I looked at the page, the textarea looks fine to me. You might be looking at an older version of the page...? I am closing this, if anything specific comes up then a new issue would be better for that specific part. 

Thanks @schalkneethling for the fix! :clinking_glasses:

**Comment by schalkneethling** [comment]
> I can confirm that the menu no longer overflows, but some content still does. The textarea at https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/HTML_text_fundamentals#active_learning_marking_up_our_recipe_page is one example

Ah yes, that is because that `iframe` has an explicit width of `900px` set on it. Perhaps we should just set a `max-width` of 100% on all iframes inside the main content. Thoughts, @chrisdavidmills?

**Comment by chrisdavidmills** [comment]
> > I can confirm that the menu no longer overflows, but some content still does. The textarea at https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/HTML_text_fundamentals#active_learning_marking_up_our_recipe_page is one example
> 
> Ah yes, that is because that `iframe` has an explicit width of `900px` set on it. Perhaps we should just set a `max-width` of 100% on all iframes inside the main content. Thoughts, @chrisdavidmills?

Yeah, I reckon that'd do it
